lib/minheap.js

```diff
@@ -11,18 +11,36 @@ class MinHeap {
   }
 
   // This method adds a HeapNode instance to the heap
-  // Time Complexity: ?
-  // Space Complexity: ?
+  // Time Complexity: O(log n)
+  // Space Complexity: O(1)
   add(key, value = key) {
```

Comment:
✨ However space complexity will be O(log n) here because of the recursive call stack of `heap_up`